crypto/rand/rand_unix.c

```diff
@@ -43,12 +43,7 @@
  * relative difference between the speed of the clock and the rate at which
  * it can be read.
  *
- * If this code is ported to an environment where execution speed is more
- * constant or where the RTC ticks at a much slower rate, or the clock can be
- * read with fewer instructions, it is likely that the results would be far
- * more predictable.
- *
- * As a precaution, we generate 4 times the minimum required amount of seed
+ * As a precaution, we generate four times the required amount of seed
```

Comment:
It's a legacy O/S that we still support.  If you want me to restore the comment I can, but I found it more misleading than not.